Test/TestCase/BaseTestCase.php

```diff
@@ -95,6 +102,17 @@ protected function setUp()
         } else {
             $this->type = new \Elastica\Type($this->index, View::INDEX_TYPE);
         }
+
+        $this->timeSeriesIndex = new \Elastica\Index($this->elasticaClient, StatusLog::INDEX_NAME . BaseTestCase::TIME_SERIES_TEST_DATE_SUFFIX);
```

Comment:
Actually we don't really need to create the index (and store extra .json files for the mappings). Our unit tests in M&E use the MappingManager::updateMappings() and MappingManager::updateTemplates() for the initial setup. Although this is fine for now since the unit tests already has this style, it will be good to make this improvement.